### PR TITLE
Add parameter freyja_depthcutoff

### PR DESCRIPTION
**Pull request by Joon-Klaps**
## PR checklist
Solves issue #405 

Freyja is throws a known error if it deals with samples of low coverage. It got fixed in this PR https://github.com/andersen-lab/Freyja/pull/145 by including the option `--depthcutoff`. 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/viralrecon/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/viralrecon _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by Joon-Klaps** [comment]
@nf-core-bot fix linting

**Comment by svarona** [comment]
Maybe you should add the changes to the `CHANGELOG.md`?

**Comment by Joon-Klaps** [comment]
> Maybe you should add the changes to the `CHANGELOG.md`?

Forgot, I'll do my other PR as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,16 +20,18 @@ Thank you to everyone else that has contributed by reporting bugs, enhancements
...
-| `--freyja_barcodes` |               |
-| `--freyja_lineages` |               |
+| Old parameter          | New parameter |
+| ---------------------- | ------------- |
+| `--skip_freyja`        |               |
+| `--freyja_repeats`     |               |
+| `--freyja_db_name`     |               |
+| `--freyja_barcodes`    |               |
+| `--freyja_lineages`    |               |
+| `--freyja_depthcutoff` |               |
```
Comment (svarona) [inline CHANGELOG.md:34]: I think --freyja_depthcutoff should go in the new parameter column?

Comment (Joon-Klaps) [reply CHANGELOG.md:34]: I thought so too 😅 but given that the previous PR, introducing freyja had its variables in the old column I putted it there.

Comment (svarona) [reply CHANGELOG.md:34]: Okay then everything looks good to me!

**Review by svarona — APPROVED** [review]

**Pull request merged**